src/command_modules/azure-cli-role/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+2.1.12
```

Comment:
Given that this is a breaking change, it seems like the bump to the version number should be bigger than a patch bump. @tjprescott indicated that the desired number should be `2.2.0`

Reply:
You are right. Will do